Dockerfile

```diff
@@ -1,35 +1,34 @@
-# Dockerfile contains instructions how to build a Docker image that will contain
-# all the code and configuration needed to run your actor. For a full
-# Dockerfile reference, see https://docs.docker.com/engine/reference/builder/
+# First, specify the base Docker image. You can read more about
+# the available images at https://sdk.apify.com/docs/guides/docker-images
+# You can also use any other image from Docker Hub.
+# The 16 represents the version of Node.js you want to use.
+FROM apify/actor-node:16
```

Comment:
Use node:14 please until we release the SDK v2 which fixes the unhandled exception crash.